### ISSUE TITLE
Add an additional arg to manager.shell --silent in keymap.toml config file

### ISSUE THREAD

**Issue by annymosse** [sxyazi/yazi#970]
### Please describe the problem you're trying to solve

By adding this feature we can run a specific shell command silently without showing up the template input.

### Would you be willing to contribute this feature?

- [ ] Yes, I'll give it a shot

### Describe the solution you'd like

Maybe showing up the input will add more flexibility for example attach a variable name to the input then use that variable inside the shell command .
In a real world example where compressing file(s) :
select file(s) -> pressing a key -> pop-up the input box -> set the extension (ex:7z,zip,...) -> press enter -> the shell receive the extension name and append it to the command and run it .

**Comment by annymosse** [comment]
> `shell --confirm` is what you're looking for, see docs here https://yazi-rs.github.io/docs/configuration/keymap/#manager.shell
> 
> Closing as already exists

I tried it but kept showing up the input field!, If so,maybe it's a bug.
thanks for your great project, I'm using it as alternative to many file managers (ex: `ranger`, `broot`,...) I already set this to @sxyazi run on startup terminal as a drop-in-replacement to GUI applications in with daily usage.

**Comment by sxyazi** [comment]
How do you use it? Could you paste your config here please? Also what's your Yazi version?

**Comment by annymosse** [comment]
@sxyazi sorry for the delay, I checked my configs this weekend and I found the root issue.
 I'm using different Terminals for different purpose and one of my terminals (WezTerm) used mainly with Yazi after switching from xplr , When I migrated my configs, folders,...etc I didn't update the shell to the new locations , and Yazi works well without crashes even by keeping she wrong locations EXCEPT it doesn't load the custom (pluggins & configs).
Note: Yazi output the issue by saying the folder doesn't exist however that time I kept work as fine for most my use cases which make me ignore the warring output.
Thanks again for your amazing program.
